content/en/blog/2022-11-11-prove-the-authenticity-of-helm-charts/index.md

```diff
@@ -0,0 +1,276 @@
+
+Starting with Helm [v3.8.0](https://helm.sh/blog/storing-charts-in-oci/), Helm supports the OCI registry as a one of the storage option for Helm charts as an alternative to Helm repositories. The [Helm CLI](https://helm.sh/docs/helm/helm/) can push and pull Helm charts to and from OCI-compliant registries.
+
+{{% note %}}
+Prior to Helm v3.8.0, OCI support was experimental and must be enabled by setting the `HELM_EXPERIMENTAL_OCI` environment variable to `1`.
+{{% /note %}}
+
+As we store Helm charts in OCI-compliant registries as OCI artifacts, we cab now use the cosign integration to sign and verify them. Also, thanks to Flux, you can reconcile resources such as plain-text Kubernetes YAML manifests, Terraform modules, etc. from OCI-compliant registries with the help of `OCIRepository` resources. You can achieve the same thing for Helm charts with [HelmRepository](/flux/components/source/helmrepositories/#helm-oci-repository) resources, which means that you can store Helm charts in OCI-compliant registries as OCI artifacts and use Flux to reconcile them like the following:
+
+```yaml
+---
+apiVersion: source.toolkit.fluxcd.io/v1beta2
+kind: HelmRepository
+metadata:
+  name: podinfo
+  namespace: default
+spec:
+  type: "oci"
+  interval: 5m0s
+  url: oci://ghcr.io/stefanprodan/charts
+```
+
+> ⚠️ Note: Here you can review the complete registries lists that support the OCI artifact specification: [OCI-Conformant Products](https://conformance.opencontainers.org/#distribution-spec)
+>
+> ⚠️ You will notice that when you open the list, DockerHub is not included into the list but it will be added soon because they recently announced OCI Artifacts support, and you can read more about it from [here](https://www.docker.com/blog/announcing-docker-hub-oci-artifacts-support/).
+
+Let's jump right into the details of how we can actually use it.
+
+We will deploy [Prometheus](https://prometheus.io/) by using its community [Helm charts](https://github.com/prometheus-community/helm-charts) stored as OCI artifacts in OCI registry. Recently, Prometheus' community started to publish their Helm charts to OCI registries and sign them with cosign using the [keyless](https://github.com/sigstore/cosign/blob/main/KEYLESS.md) approach, you can learn more the process [here](https://github.com/prometheus-community/helm-charts/pull/2631). Then we are going to verify it with _cosign_ and configure Flux to verify the Helm chart' signatures before they are downloaded and reconciled. As Prometheus' community signed their Helm Charts without providing a key pair, we do not need to specify any key in the HelmChart resource' `provider.cosign` spec to enable keyless verification for Flux.
+
+> For the sake of simplicity, we've deployed Prometheus alone but if you want to learn more about installing the Prometheus stack including Grafana, Alertmanager, etc., please refer to the official Flux [page](#TBD) that can help you to do that.
```

Comment:
`TBD`

Reply:
yep, @stefanprodan will give us the correct page.

Reply:
Here it is https://fluxcd.io/flux/guides/monitoring/